homeassistant/components/honeywell/climate.py

```diff
@@ -201,6 +203,10 @@ def device_state_attributes(self) -> Dict[str, Any]:
         data[ATTR_FAN_ACTION] = "running" if self._device.fan_running else "idle"
         if self._device.raw_dr_data:
             data["dr_phase"] = self._device.raw_dr_data.get("Phase")
+        if self._device.outdoor_temperature:
+            data[ATTR_OUTDOOR_TEMP] = self._device.outdoor_temperature
+        if self._device.outdoor_humidity:
+            data[ATTR_OUTDOOR_HUMIDITY] = self._device.outdoor_humidity
```

Comment:
These should not be attributes, but separate sensors.

Reply:
Gotcha; I wasn't sure because all the other data from the thermostat is returned via attributes in the main climate entity; I'll refactor this to create 2 new sensors instead.

Reply:
It looks like in order to do that properly I'll need to convert this integration from a single platform one to a multi platform one, which would necessarily involve updating the configuration.yaml. Since it looks like configuration.yaml entries are being depreciated I may as well convert this over to use a config flow at that time.

That's a much bigger change than this original PR though and TBH I'm not sure when I'll be able to create and submit those changes; should I close this out and create a new PR when I'm ready or just leave this open and update it when I'm able to?

Reply:
That is up to you. Closing sounds fine though, starting fresh 👍